feature/experimental/p4rt/README.md

```diff
@@ -10,12 +10,26 @@ This document specifies the requirements for p4rt test implementation.
     1.  WBB P4 Protobuf file:
         https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/p4rt/wbb.p4info.pb.txt
 
-3.  The client should make use of Ondatra Raw API
-    `dut.RawAPIs().P4RT().Default(t)`
+3.  Tests should instantiate new P4RT clients using
+    `p4rt_client.NewP4RTClient()` function. This instantiates the
+    `StreamTermErr` channel required to investigate errors when the p4rt stream
+    terminates.
 
-4.  Tests should get the P4RT Node Name by walking the Components OC tree.
+4.  Tests should make use of Ondatra Raw API `dut.RawAPIs().P4RT().Default(t)`
+    during client instantiation.
+
+5.  To avoid indefinite blocking during Get response
+    `client.StreamChannelGetArbitrationResp()`, tests should make use of
+    `p4rtutils.StreamTermErr()` function which returns an Error if the p4rt
+    stream has been terminated.
```

Comment:
Let's explain why the indefinite blocking happens, namely that GetArbitrartion() blocks on the cond.Wait() after the cstream Go routine exits, which could be fixed by replacing cond.Wait() with a sleep (or a select on StreamTermErr channel).

Let's make it clear that the suggested workaround here does not fix the underlying issue. Use a code example to illustrate what the workaround looks like.